ccm_web/server/src/lti/lti.service.ts

```diff
@@ -19,6 +19,9 @@ const createLaunchErrorResponse = (res: Response, action?: string): Response =>
   return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json(message)
 }
 
+// the array of user roles that are allowed to use CCM tool
+const allowedRoles = ['TeacherEnrollment', 'TaEnrollment', 'ObserverEnrollment', 'DesignerEnrollment', 'Account Admin', 'Sub-Account Admin']
+
```

Comment:
Are these roles defined anywhere else in the backend? I'm not exactly sure where the best place to put them is. This may be okay for now, but if we ever needed these roles in other parts of the application (like for role-based authorization), we'd need to re-think. Other places we might put them? `canvas.interfaces.ts`? I think these strings are Canvas specific, not part of the LTI spec. Let me know your thoughts. I'm not asking for any changes yet.

Reply:
if the canvas.interfaces.ts is the central place for defining const and interfaces for server side code, then this definition could move there as another enum: AllowedUserEnrollmentType?

Reply:
I see some of the these roles already mentioned in the `canvas.interface.ts`
```
export enum UserEnrollmentType {
  DesignerEnrollment = 'DesignerEnrollment',
  ObserverEnrollment = 'ObserverEnrollment',
  StudentEnrollment = 'StudentEnrollment',
  TaEnrollment = 'TaEnrollment',
  TeacherEnrollment = 'TeacherEnrollment'
}
```
I think we can create a new interface

```
export enum LTIRole extend UserEnrollmentType {
  AccountAdmin = 'Account Admin'
  SubaccountAdmin = 'Sub-Account Admin'
  Support Consultant = 'Support Consultant'
}
```

Reply:
I have updated the lti role definitions into the canvas.interfaces.ts. Please review

Reply:
Considering this one resolved since you moved the role definitions.